### ISSUE TITLE
Offlining former master node fails

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1095]
Originally reported of Google Code with ID 1039.

```
<b>What steps will reproduce the problem?</b>
1. Create a three-node cluster.
2. Shutdown the current master.
3. Do a master-failover on another node.
4. Try to set the former master as offline (modify -O yes)

<b>What is the expected output? What do you see instead?</b>

The operation should succeed with a few warnings, but instead it fails after stating that the SSH setup cannot be adjusted. As the node is presumably offline, this attempt should not be made.

The configuration is modified correctly, so this is not too big a flaw, but it is wrong for the operation to fail.
```
Originally added on 2015-03-13 13:55:53 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
What Ganeti version is that?
```
Originally added on 2015-03-17 09:39:38 +0000 UTC.

Added Labels: Type-Defect Priority-Medium

**Comment by Ganeti-Issues-Migrator** [comment]
```
2.15.0
```
Originally added on 2015-03-17 10:09:52 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2015-03-20 13:28:17 +0000 UTC.

Changed State: Started
Added to Milestone: Release2.13

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed with:

commit eee010979b19dc58f4270c8f2bca97db157cd8f2
Author: Helga Velroyen <helgav@google.com>
Date:   Fri Mar 20 11:42:50 2015 +0100

    Be gentler to failures while removing SSH keys
    
    When removing SSH keys, it can happen that updating the SSH
    files fails on some nodes. This patch makes sure that we
    collect the failures of the nodes and display them to the
    user instead of breaking as soon as the first occurs.
    This fixes Issue 1039.
    
    Signed-off-by: Helga Velroyen <helgav@google.com>
    Reviewed-by: Petr Pudlak <pudlak@google.com>

```
Originally added on 2015-03-24 10:41:11 +0000 UTC.

Mentioned Issue with GoogleCode ID 1039 has been migrated to GitHub with ID #1095.
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.14.0
```
Originally added on 2015-06-02 10:37:03 +0000 UTC.

Changed State: Released
